components/docs-chef-io/content/automate/ha_backup_restore_prerequisites.md

```diff
@@ -13,7 +13,10 @@ gh_repo = "automate"
     weight = 210
 +++
 
-This page explains the prerequisites of the AWS and on-premise backup.
+
+{{< note >}} For AWS infra, if you added `backup_config` during provisioning, prerequistes will already configured as part of deployment itself. So this can be skipped if you are using AWS infra. {{< /note >}}
```

Comment:
REPLACE IT WITH: For AWS infra, if you added `backup_config` during provisioning, prerequisites will already be configured as part of deployment itself. Skip this step if you are using AWS infra.

Reply:
Done